docs/references/accessibility-tools.md

```diff
@@ -67,28 +67,80 @@ Note that `Narrator` key is setup by default as being `Caps Lock` or `Insert` ke
 
 ### High contrast
 
-To test High Contrast, please use the High Contrast flag offered directly in your browser when it exist over any pluggins.
+To test high contrast theme, please use the high hontrast flag offered directly in your browser when it exist over any pluggins.
 
 #### Edge
 
-Go to the Edge forced-colors flag (**edge://flags/#forced-colors**), enable the experiment and restart your browser.
-Then open Windows High Contrast settings, turn it on and choose which HC you want to test.
+Find [Edge forced-colors flag](edge://flags/#forced-colors), enable the experiment and restart your browser.
+Then open Windows high contrast theme settings, turn it on and choose which HC you want to test.
 
 #### Chrome
 
-Go to the Chrome forced-colors flag (**chrome://flags/#forced-colors**), enable the experiment and restart your browser.
-Then open Windows High Contrast settings, turn it on and choose which HC you want to test.
+Find [Chrome forced-colors flag](chrome://flags/#forced-colors), enable the experiment and restart your browser.
+Then open Windows high contrast theme settings, turn it on and choose which HC you want to test.
 
 #### Firefox
 
-Firefox works with Windows High Contrast. It automatically detects if you are using a HC theme and displays everything in your HC color scheme.
+Firefox works with Windows high contrast theme. It automatically detects if you are using a HC theme and displays everything in your HC color scheme.
 
-Just make sure Firefox override colours only with High Contrast themes.
-To check that, go on Firefox settings and click on the `Colours…` button under ‘Language and Appearance’ section and make sure that `Only with High contrast themes` option is selected for the color override.
+Just make sure Firefox override colours only with high contrast themes.
+To check that, go on Firefox settings and click on the ‘Colours…’ button under ‘Language and Appearance’ section and make sure that 'Only with High contrast themes" option is selected for the color override.
 
 #### Safari
 
 You will need to use the display pane of Accessibility Display preferences and use the `Invert colors` option.
 
 To change these preferences, go to Apple menu > System Preferences, click on Accessibility, then Display and finally Display 
```

Comment:
This could change between macos releases, can we instead link to an apple accessibility guide for enabling this?

Reply:
Perhaps: https://support.apple.com/lv-lv/guide/mac-help/unac089/mac